concourse/pipeline.yml

```diff
@@ -0,0 +1,149 @@
+health_status_notify: &health_status_notify
+  put: health-notification
+
+resource_types:
+  - name: http-api
+    type: docker-image
+    source:
+      repository: gdscyber/http-api-resource
+
+resources:
+  - name: ggroup-data-to-splunk-git
+    icon: github-circle
+    type: git
+    source:
+      uri: https://github.com/alphagov/cyber-security-ggroup-data-to-splunk.git
+      branch: concourse
+
+  - name: health-notification
+    type: http-api
+    source:
+      uri: https://((health_host_prod))/?alert_type=concourse$&alert_name=health
```

Comment:
Is the `$` supposed to be in the URL?

Reply:
Looking at other pipelines that use this health notification - yes

Reply:
Great!